doc/changelog/03-notations/15754-changelog_15754.rst

```diff
@@ -0,0 +1,12 @@
+- **Removed:**
+  ``_`` in ``ident`` entries in notations, which was deprecated
+  in favor of ``name`` in 8.13. When you see messages like
+  ``Error: Notation "[ rel _ _ : _ | _ ]" is already defined at level 0
+  with arguments name, name, constr, constr while it is now required to be
+  at level 0 with arguments ident, ident, constr, constr.``,
```

Comment:
It would be a nice touch to include this message in the doc somewhere.  Probably on the Notation command.

FWIW "while it is now" is not well worded ("now" is vague), but not your handiwork.

Reply:
Indeed, this error seems undocumented.